ejdb-sys/Cargo.toml

```diff
@@ -11,6 +11,11 @@ build = "build.rs"
 
 [build-dependencies]
 pkg-config = "0.3"
+cmake = "0.1"
+bindgen = "0.46.0"
 
 [dependencies]
 libc = "0.2"
+
+ [lib]
+ doctest = false
```

Comment:
Indentation seems off.

Reply:
No, it's just diff makes it look like this =)

Reply:
Hmm, that's strange. Didn't see this in my work's GH enterprise instance.